src/main/java/net/glowstone/entity/projectile/GlowEnderPearl.java

```diff
@@ -0,0 +1,83 @@
+package net.glowstone.entity.projectile;
+
+import static net.glowstone.net.message.play.entity.SpawnObjectMessage.THROWN_ENDERPEARL;
```

Comment:
I would prefer using `SpawnObjectMessage.THROWN_ENDERPEARL` in line.

Reply:
Done.